frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts

```diff
@@ -37,6 +37,7 @@ import {DebouncedRequestSwitchmap, errorNotificationHandler} from "core-app/help
 import {HalResourceNotificationService} from "core-app/modules/hal/services/hal-resource-notification.service";
 import {NgSelectComponent} from "@ng-select/ng-select";
 import {UserResource} from "core-app/modules/hal/resources/user-resource";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
 export const usersAutocompleterSelector = 'user-autocompleter';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/cfe89dd706c72154281f341454501767d598e770/frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts#L43"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>